templates/login_or_signup.tmpl

```diff
@@ -5,17 +5,105 @@
         <meta charset="utf-8" />
         <meta
             name="description"
-            content="The best way to book club" />
+            content="A better way to book club" />
         <meta
             name="viewport"
             content="width=device-width, initial-scale=1" />
 
         <title>CerealNotes</title>
 
-        <script type="text/javascript" src="/static/js/login_or_signup.js"></script>
+        <link href="//cdn.muicss.com/mui-0.9.36/css/mui.min.css" rel="stylesheet" type="text/css" />
+        <link href="/static/css/login_or_signup.css" rel="stylesheet" type="text/css" />
+
+        <script src="//cdn.muicss.com/mui-0.9.36/js/mui.min.js"></script>
+        <script src="//ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+        <script src="/static/js/login_or_signup.js"></script>
     </head>
 
     <body>
-        Hello World
+        <div class="mui-container">
+            <h1 class="mui--text-center">
+                CerealNotes
+            </h1>
+
+            <div class="mui-row">
+                <div class="mui-col-sm-6 mui-col-md-4 mui-col-sm-offset-3 mui-col-md-offset-4">
+                    <ul class="mui-tabs__bar mui-tabs__bar--justified">
+                        <li class="mui--is-active">
+                            <a data-mui-toggle="tab" data-mui-controls="signup-form">
+                                Signup
+                            </a>
+                        </li>
+
+                        <li>
+                            <a data-mui-toggle="tab" data-mui-controls="login-form">
+                                Login
+                            </a>
+                        </li>
+                    </ul>
+
+                    <br />
+
+                    <div id="signup-form" class="mui-tabs__pane mui--is-active">
+                        <div class="mui-form">
+                            <div class="mui-textfield mui-textfield--float-label">
+                                <input
+                                    type="text"
+                                    name="displayName"
+                                    maxlength="128"
+                                    required
+                                />
+                                <label>Name</label>
+                                <span class="validation-message mui--text-accent-secondary"></span>
+                            </div>
+
+                            <div class="mui-textfield mui-textfield--float-label">
+                                <input
+                                    type="email"
+                                    name="emailAddress"
+                                    maxlength="128"
```

Comment:
why add max? is there some restriction based on some tech we use?

Reply:
yeah i don't have a strong reason. i could be convinced to remove. my only thought was that it seems unlikely that anything over this length would be a real user